java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -194,7 +194,11 @@ private synchronized Path getBinary() {
         } else if (current.is(MAC)) {
           folder = "macos";
         } else if (current.is(LINUX)) {
+          if (System.getProperty("os.arch").contains("arm")) {
+            throw new WebDriverException("Linux ARM is not supported by Selenium Manager");
+          } else {
           folder = "linux";
```

Comment:
can we check the formatting of this conditional

Reply:
fixed